### PR TITLE
React native setup

### PR DESCRIPTION
**Pull request by karengpalomo**
React native application for caregivers was set up. This is intended to be used on mobile.

### CODE REVIEW

File: .DS_Store
Comment (imharris2702) [inline .DS_Store:1]: Don't think we want to include this file, can we remove it from the branch?

**Review by imharris2702 — CHANGES_REQUESTED** [review]
Looks like a .DS_Store got added in on accident, needs to be removed

File: .DS_Store
Comment (karengpalomo) [reply .DS_Store:1]: Yeah i can go ahead and delete it so that it's removed from the branch altogether

**Review by imharris2702 — APPROVED** [review]
looks good to me

**Pull request merged**